competencies/templates/competency_difficulty_form.html

```diff
@@ -0,0 +1,64 @@
+{% load static %}
+
+<!DOCTYPE html>
+<html>
```

Comment:
Missing `lang="en"` for validation purposes.

But why are we creating a new page instead of extending the main base.html template?

Reply:
Added as suggested.